tests/src/core/testqgsmesheditor.cpp

```diff
@@ -20,6 +20,7 @@
 #include "qgstriangularmesh.h"
 #include "qgsmeshlayer.h"
 #include "qgsmesheditor.h"
+#include "qgsmeshadvancedediting.h"
```

Comment:
do we have tests for Delaunay triangulation too?

Reply:
I think it would be good to have unit tests for the `QgsMeshUtils` for each function to test edge cases.

Reply:
not directly, but the classes that are used for the triangulation are already tested. I will look to add specific test for here.